### PR TITLE
CSS loading fix

### PR DESCRIPTION
**Pull request by wojtekw92**
## What is the purpose of this pull request?

Bug fix

## Does your PR contain necessary tests?

All patches which change the editor code must include tests. You can always read more
on [PR testing](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_contributing_code-section-tests),
[how to set the testing environment](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests) and
[how to create tests](https://docs.ckeditor.com/ckeditor4/docs/#!/guide/dev_tests-section-creating-your-own-test)
in the official CKEditor documentation.

### This PR contains

- [X] Unit tests
- [X] Manual tests

## What changes did you make?

CSS loading fix in inlinetoolbar.

### CODE REVIEW

File: tests/plugins/inlinetoolbar/manual/skins.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	<p>Apollo 11</p>
+	<p>Apollo 11</p>
+	<p>Apollo 11</p>
+</textarea>
+
+<script>
+	CKEDITOR.disableAutoInline = true;
+
+	function instanceReadyListener( evt ) {
+		var image = this.editable().findOne( 'img' ),
```
Comment (mlewand) [inline tests/plugins/inlinetoolbar/manual/skins.html]: This variable is not used.

File: tests/plugins/inlinetoolbar/manual/skins.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+@bender-ui: collapsed
+@bender-tags: 4.8.0, feature, inlinetoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,notification,floatingspace,inlinetoolbar,sourcearea,list,link,image,language,stylescombo
+
+Check in browser console, if there is no errors with CSS loading.
```
Comment (mlewand) [inline tests/plugins/inlinetoolbar/manual/skins.md]: > if there ~is~**are** no errors

Comment (mlewand) [reply tests/plugins/inlinetoolbar/manual/skins.md]: I have have runed this manual test.

And it **does throw** an error related to CSS loading:

```
GET http://tests.ckeditor.test:1030/apps/ckeditor/plugins/balloonpanel/skins/kama,/apps/ckeditor/skins/kama//balloonpanel.css net::ERR_ABORTED
```

File: tests/plugins/inlinetoolbar/cssloading.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+	'use strict';
+
+	bender.editor = {
+	};
+	CKEDITOR.skinName = 'kama,/apps/ckeditor/skins/kama/';
+	var spy = sinon.spy( CKEDITOR.dom.document.prototype, 'appendStyleSheet' );
+	bender.test( {
+		'test loading css with path': function() {
+			assert.areSame( 'kama,/apps/ckeditor/skins/kama/', CKEDITOR.skinName, 'config skinName should be with path' );
+			sinon.assert.calledWith( spy, 'http://127.0.0.1:1030/apps/ckeditor/plugins/inlinetoolbar/skins/kama/inlinetoolbar.css' );
```
Comment (mlewand) [inline tests/plugins/inlinetoolbar/cssloading.js]: This test case fails for me.

```
Unexpected error: expected appendStyleSheet to be called with arguments http://127.0.0.1:1030/apps/ckeditor/plugins/inlinetoolbar/skins/kama/inlinetoolbar.css
    appendStyleSheet(/apps/ckeditor/skins/kama/editor.css) at loadCss (http://tests.ckeditor.test:1030/apps/ckeditor/core/skin.js:168:22)
    appendStyleSheet(http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/skins/default.css) at Object.beforeInit (http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/plugin.js:530:23)
    appendStyleSheet(http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/skins/kama/inlinetoolbar.css) at Object.beforeInit (http://tests.ckeditor.test:1030/apps/ckeditor/plugins/inlinetoolbar/plugin.js:531:23)
    appendStyleSheet(http://tests.ckeditor.test:1030/apps/ckeditor/plugins/balloonpanel/skins/kama,/apps/ckeditor/skins/kama//balloonpanel.css) at Object.init (http://tests.ckeditor.test:1030/apps/ckeditor/plugins/balloonpanel/plugin.js:20:23)
Expected: undefined (undefined)
Actual:   undefined (undefined)
```

Our standard test domain is test `tests.ckeditor.test` but you should pick it dynamically.

File: tests/plugins/inlinetoolbar/manual/skins.md (hunk 2 above)
Comment (wojtekw92) [reply tests/plugins/inlinetoolbar/manual/skins.md]: this one is fixed by #1221

Comment (mlewand) [reply tests/plugins/inlinetoolbar/manual/skins.md]: I know, but here on branch it fails. You need to write exactly what is the expected result.

Person that views it doesn't know that it's fixed by #1221.

**Pull request merged**